src/resources/Incorporations/limitedCompany.ts

```diff
@@ -39,6 +39,11 @@ export const LimitedCompanyResource: IncorporationResourceIF = {
   shareClasses: {
     countMinimum: 1
   },
+  incorporationArticles: {
+    articles: '<br>BC LTD Articles',
```

Comment:
Why do some of these start with a line break, a space, or nothing at all? This seems risky (another dev might try to remove extra spaces, etc). If a leading space/br is needed, can it be added in the template?

Reply:
OK, I realize you just move the code that was there previously. But I can't think of a reason why this should be different for each entity type. The screenshots should help clarify this.

Reply:
Yes the extra spaces were a leftover and not necessary I can remove them. The line break prevents BC LTD Articles from getting separated on two lines which then causes the tooltip to be centered on the page instead of over the text. The other articles don't have this problem. I can add a comment so people know why it is there.

Reply:
Works for me. Thanks for the explanation.